### PR TITLE
[2 of 3]: DCOS-11542: Independent scrolling of form and JSON editor in ServiceCreateModal

### PR DESCRIPTION
**Pull request by jfurrow**
~~This depends on #1567.~~

This PR separates the scrolling of the two containers.

### CODE REVIEW

**Comment by rcorral** [comment]
Why the hold merge @jfurrow ?

File: src/js/components/Page.js [hunk 1]
```diff
@@ -80,7 +80,7 @@ var Page = React.createClass({
   },
 
   onSidebarStoreWidthChange() {
-    GeminiUtil.updateWithRef(this.refs.gemini);
+    ScrollbarUtil.updateWithRef(this.refs.gemini);
```
Comment (rcorral) [inline src/js/components/Page.js]: Can you implement `FluidGeminiScrollbar` in this file too?

Comment (jfurrow) [inline src/js/components/Page.js]: Yep! I have another PR coming for that :)

**Comment by jfurrow** [comment]
@rcorral I added `hold merge` until #1567 merges

**Comment by nLight** [comment]
Clean work! 👍
Even though I'm not a fan of hacking with adding elements in <head />

**Comment by orlandohohmeier** [comment]
I get an error and the don't see the JSON editor anymore... 

```
Uncaught TypeError: _ScrollbarUtil2.default.getScrollbarWidth is not a function(…)
```

**Comment by orlandohohmeier** [comment]
Rebased the branch and fixed the `updateWithRef` util which cause the error and was the reason why the JSON editor wasn't rendering.

**Comment by jfurrow** [comment]
Thanks for the rebase and the bug fix @orlandohohmeier.

**Pull request merged**